thinc/api.py

```diff
@@ -12,7 +12,13 @@
 from .schedules import decaying, slanted_triangular, compounding
 from .types import Ragged, Padded, ArgsKwargs, Unserializable
 from .util import fix_random_seed, is_cupy_array, set_active_gpu
-from .util import prefer_gpu, require_gpu, require_cpu, DataValidationError, data_validation
+from .util import (
+    prefer_gpu,
+    require_gpu,
+    require_cpu,
+    DataValidationError,
+    data_validation,
+)
```

Comment:
```suggestion
from .util import prefer_gpu, require_gpu, require_cpu,
from .util import DataValidationError, data_validation
```
Battling `black` is like a full-time job ;-)